tvm/src/codegen/build_common.h

```diff
@@ -34,7 +34,8 @@ ExtractFuncInfo(const Array<LoweredFunc>& funcs) {
 enum OutputMode : int {
   HostDevice = 0,
   HostOnly   = 1,
-  DeviceOnly = 2
+  DeviceOnly = 2,
+  VHLSDevice = 3
```

Comment:
Why do we have `VHLSDevice` here?

Reply:
It's used to generate VHLS kernel code. See the below comment.